README.md

```diff
@@ -45,9 +45,10 @@ class Foo
 end
 ```
 
-If `#as_json` is called without the `:presenter_action` options, or if the
-presenter class doesn't exist, this delegates to the original model class
-definition of `#as_json`
+If `#as_json` is called without the `:presenter_action` options, or if
+the presenter class doesn't exist, this delegates to
```

Comment:
Oh I see from your other PR that you are passing in unimplemented `:presenter_action`s and depending on the fallback to `#as_default_json`. That seems a little confusing to me. Once you know how this gem works, if you see a `:presenter_action`, you expect to find the matching method implemented. I'm not sure it buys us anything to be able to pass in bogus `:presenter_actions` unless you want to be able to do something like pass in the controller action and let the presenter decide if there's a matching implementation.